Classes/Dimension/ConfigurationBasedContentDimensionSource.php

```diff
@@ -22,25 +24,24 @@ final class ConfigurationBasedContentDimensionSource implements ContentDimension
     const CONSTRAINT_IDENTIFIER_WILDCARD = '*';
 
     /**
-     * @var array
+     * @var array<string,mixed>
      */
-    protected $dimensionConfiguration;
+    protected array $dimensionConfiguration;
```

Comment:
Why not `private`?

Reply:
Probably because I forgot to change it :)